### PR TITLE
Display course modes in authoring tool.

### PR DESCRIPTION
**Pull request by jimabramson**
ECOM-3635

@AlasdairSwan @schenedx FYI.

### CODE REVIEW

**Comment by schenedx** [comment]
@jimabramson Could you put a screenshot in the PR so I can see the UI change?

**Comment by jimabramson** [comment]
@schenedx  hyg:
screen shot 2016-02-18 at 8 31 20 am
FYI @marcotuts

**Comment by schenedx** [comment]
:+1:

File: programs/static/templates/course_run.underscore [hunk 1]
```diff
@@ -21,6 +21,10 @@
         gettext('Start Date: %(date)s'),
         { date: start_date },
         true
+    ) %><br/><%= interpolate(
```
Comment (AlasdairSwan) [inline programs/static/templates/course_run.underscore]: I don't like the ```<br/>```, could we use CSS or a new ```<p>``` to create the same effect?

**Comment by jimabramson** [comment]
changed to use two divs instead of a p with a br.   looks a little nicer IMO
screen shot 2016-02-19 at 10 50 23 am

**Comment by AlasdairSwan** [comment]
Once tests pass :+1:

**Pull request merged**